docs/ECCTOOL_EXAMPLES.md

```diff
@@ -8,12 +8,13 @@ In this example, we will use `ecctool repairs` to check the status of manual rep
 The output shows all manual repairs for all ecChronos instances.
 
 ```bash
----------------------------------------------------------------------------------------------------------------------------------------------------
-| Id                                   | Host Id                              | Keyspace | Table  | Status    | Repaired(%) | Completed at        |
----------------------------------------------------------------------------------------------------------------------------------------------------
-| f4fb2b38-b9d0-4390-97ca-eeb284391f80 | ee32d9c7-1a4e-40c2-9b28-1000544011ae | test     | table1 | IN_QUEUE  | 0.00        | -                   |
-| f4fb2b38-b9d0-4390-97ca-eeb284391f80 | ba7665b2-5a7b-42b5-9f38-037f2da1e80a | test     | table1 | COMPLETED | 100.00      | 2022-09-22 13:40:07 |
----------------------------------------------------------------------------------------------------------------------------------------------------
+-----------------------------------------------------------------------------------------------------------------------------------------------------------------
+| Id                                   | Host Id                              | Keyspace | Table  | Status    | Repaired(%) | Completed at        | Repair type |
+-----------------------------------------------------------------------------------------------------------------------------------------------------------------
+| f4fb2b38-b9d0-4390-97ca-eeb284391f80 | ee32d9c7-1a4e-40c2-9b28-1000544011ae | test     | table1 | IN_QUEUE  | 0.00        | -                   | VNODE       |
```

Comment:
What do you think of having two tables under the same command?

Two tables that are printed out by the command. One for incremental one for vnode. If either incremental or vnode doesn't exist we could just not print it. This way we don't get a long list of repair types. What do you think?

Reply:
Hmm, i need to think about this.

Reply:
I think having 2 tables will be confusing for the users if --limit flag is used. Maybe we should add --incremental --vnode flags to "schedules" and "repairs" to filter on repair type, wdyt?

Reply:
I understand what you mean but unfortunately that wouldn't really fix the readability of it all for new users. I agree that limit does make splitting a bit of an issue and I can accept having flags instead to filter on repair type.